druid/README.md

```diff
@@ -0,0 +1,120 @@
+# Agent Check: druid
+
+## Overview
+
+The Datadog Agent collects many metrics from _Druid_ via DogStatsD. Those Druid metrics are related to queries, ingestion, and coordination:
+
+* Query Metrics
```

Comment:
Since we refer the reader to the metric section, this list can be removed

Reply:
fixed